api/app/utils/esri.py

```diff
@@ -0,0 +1,72 @@
+"""
+Functions for talking to ESRI ARC servers.
+"""
+import urllib.parse
+import urllib.request
+import json
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def fetch_object_list(url: str):
+    """
+    Fetch object list from a feature layer.
+
+    url: layer url to fetch
+    (e.g. https://maps.gov.bc.ca/arcserver/rest/services/whse/bcgw_pub_whse_legal_admin_boundaries/MapServer/2)
+    """
+    logger.info('fetching object list for %s...', url)
+
+    params = {
+        'where': '1=1',
```

Comment:
is this `where` clause necessary?

Reply:
Unfortunately yes. There's no "*" equivalent. There's no direct way of asking for everything, so that hack is to pass a condition that is true for all records.

Reply:
Oh - again - you want a comment in the code?

Reply:
Gotcha. (Very unusual choice on their part to require a where clause though...)

Reply:
Yeah I think a comment there would be wise.